src/components/atoms/Tile/Tile.tsx

```diff
@@ -0,0 +1,13 @@
+import { ITileProps } from '@app/components/atoms/Tile/@types/Tile';
+import Link from 'next/link';
+
+const Tile = ({
+  content,
+  link
+}: ITileProps) => (
```

Comment:
It's better to type everything as `React.FC`

```suggestion
const Tile: React.FC<ITileProps> = ({
  content,
  link
}) => (
```